package.json

```diff
@@ -117,6 +115,7 @@
     "@types/minimatch": "^3.0.1",
     "@types/node": "^8.10.38",
     "@types/semver": "^5.5.0",
+    "@types/webpack": "^4.4.19",
     "chai": "^4.2.0",
```

Comment:
It looks that `@types/webpack` moves to being a `devDependency` here. But it looks like we're using `any` in the codebase. Or is that not always the case? (I'm on my phone so I may have missed something)

Reply:
We're using it in a few dozen other places internally. These two declarations were just the only two that were exposed externally, and thus the only ones to end up in index.d.ts upon compilation.